### PR TITLE
[RFR] Added “RxJS” entry

### PR DESCRIPTION
**Pull request by scouten**

### CODE REVIEW

File: glossary/RXJS.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
 # RxJS
 
-[RxJS](http://reactivex.io/) section to be completed.
+[RxJS](https://github.com/Reactive-Extensions/RxJS) is a library in the [ReactiveX](http://reactivex.io/) family of functional reactive programming libraries. It allows you to develop programs that respond to asynchronous events by composing sequences of observable streams.
+
+Observable streams can send zero or more "data" notifications and terminate with either an "completed" or an "error" notification. Subsequent streams can combine and remix these notifications into new observable patterns, making it possible to compose rich sequences of asynchronous events that clean up resources appropriately when done.
```
Comment (KittyGiraudel) [inline glossary/RXJS.md]: Any chance you could replace dumb quotes with smart quotes here? :)

Comment (scouten) [inline glossary/RXJS.md]: Sure. Stand by.

Comment (scouten) [inline glossary/RXJS.md]: Updated PR.

**Comment by KittyGiraudel** [comment]
Thank you! :)

**Pull request merged**